sympy/printing/pretty/pretty.py

```diff
@@ -1845,7 +1851,9 @@ def _print_Pow(self, power):
                 return prettyForm("1")/self._print(b)
             n, d = fraction(e)
             if n is S.One and d.is_Atom and not e.is_Integer and self._settings['root_notation']:
-                return self._print_nth_root(b, e)
+                dp = self._print(d)
+                if dp.height() == 1:
```

Comment:
Are there tests for when height != 1?

Reply:
Not that I know of...  suppose `Catalan` decided to pretty print itself like
```
 ata
C   Lan
```